eogrow/utils/testing.py

```diff
@@ -300,6 +300,78 @@ def check_pipeline_logs(pipeline: Pipeline) -> None:
     assert load_names(pipeline.storage.filesystem, finished_filename), "No executions finished"
 
 
+def run_config(
+    config_path: str,
+    *,
+    output_folder_key: Optional[str] = None,
+    reset_output_folder: bool = True,
+) -> None:
+    """Runs a pipeline (or multiple) and checks the logs that all the executions were successful
+
+    :param config_path: A path to the config file
+    :param output_folder_key: Type of the folder containing results of the pipeline, inferred from config if None
+    :param reset_output_folder: If True it will delete content of the folder with results before running the pipeline
```

Comment:
```suggestion
    :param reset_output_folder: Delete the content of the results folder before running the pipeline. Default is `True`.
```

Reply:
defaults are kinda visible :thinking:  if you remove the last sentence i'm ok with it